### DISCUSSION TITLE
Allow per row actions for table

### DISCUSSION THREAD

**Discussion by mturoci** [h2oai/wave#633] [Ideas]
via @goldentom42 
#### Is your feature request related to a problem? Please describe

This feature would allow app devs to perform actions like delete / edit per row.

#### Describe the solution you'd like

Provide a `button_cell` type.

**Comment by rasika** [comment]
this seems to be a valid case. I also came-across this limitation

**Reply by zillionare** [reply]
I'd like the ability to edit/delete per row

**Reply by mturoci** [reply]
See https://github.com/h2oai/wave/pull/1324 available in next release (or today in nightly).
